### ISSUE TITLE
Incomplete set of tags, contact server operator

### ISSUE THREAD

**Issue by RustBandit** [Flaxbeard/ImmersivePetroleum#195]
I cannot get this mod to load in my server, but it runs just fine in my singleplayer, I am the server owner so disregard the end state in the title of this issue, please help me with whatever I'm doing wrong.

**Comment by mindforger** [comment]
You are even doing the report wrong, you did not provide any Info about your setup or any crash report. Without this it is hard to grasp your issue

**Comment by TwistedGate** [comment]
Indeed, i need a _crash-report_ or a _log file_ uploaded to [Pastebin](https://pastebin.com) or [Gist](https://gist.github.com/)
`I cannot get this mod to load in my server` does not help

**Comment by kzdaulet** [comment]
> Indeed, i need a _crash-report_ or a _log file_ uploaded to [Pastebin](https://pastebin.com) or [Gist](https://gist.github.com/)
> `I cannot get this mod to load in my server` does not help

After update i have trouble too, https://pastebin.com/1V2izknU

**Comment by TwistedGate** [comment]
> > Indeed, i need a _crash-report_ or a _log file_ uploaded to [Pastebin](https://pastebin.com) or [Gist](https://gist.github.com/)
> > `I cannot get this mod to load in my server` does not help
> 
> After update i have trouble too, https://pastebin.com/1V2izknU

To anyone reading that... it's a completely different problem. See #196
